src/app/routes/getInitialData/mediaAssetPage/index.test.js

```diff
@@ -12,20 +12,55 @@ baseUrl.mockImplementation(() => mockBaseUrl);
 jest.mock('../utils/fetchData', () => jest.fn());
 fetchData.mockImplementation(() => mockData);
 
+const defaultServiceParam = 'pidgin';
```

Comment:
don't think we use these variables, other than to move them into an object later?

Could just create the object here